tulflow/harvest.py

```diff
@@ -83,8 +84,11 @@ def harvest_oai(**kwargs):
     logging.info("Harvesting from %s", oai_endpoint)
     logging.info("Harvesting %s", harvest_params)
     request = Sickle(oai_endpoint, retry_status_codes=[500,503], max_retries=3)
-    data = request.ListRecords(**harvest_params)
-    return data
+    try:
+        return request.ListRecords(**harvest_params)
+    except NoRecordsMatch:
+        logging.info("No records found.")
```

Comment:
@ebtoner is it OK, if when no records are found we just log "No records found." instead of erring out or is there some other type of reporting you would prefer.

Reply:
Would this just show up in the harvest log at the point where the harvest is attempted? If so, I'm fine with that as a starting point, though I'm concerned that that information will get buried in a harvest with lots of records. 

@htomren See the comments/questions above. What do you think? Ideally, it would be nice to have some additional logging at the bottom of harvest task (i.e. where info like "OAI Records Harvested & Processed" is), but I'm not sure that it necessarily needs to be the full list of set identifiers. Would it be simpler to do a count or something more general?

Reply:
@dkinzer @ebtoner I agree that's a good starting point. 

In addition to that, though, I agree with emily that some kind of summary at the bottom of the log would be helpful. 
I like the idea of a count, like:
sets with no records: 1
then if we saw that we could ctrl+f to find the relevant sets. 
or, this could be
sets with no records: [list of setspecs with no records]
whichever is easier

Reply:
OK, I'll just update this to also keep track of count in the report since that's what we are doing for the other counts we are keeping track of.